### ISSUE TITLE
endKey throws an error

### ISSUE THREAD

**Issue by SSaulenas** [BalticAmadeus/AblFormatter#149]
Example:

```
      do while true on endkey undo, leave:
        create bbuffer.
      end.
```

![image](https://github.com/user-attachments/assets/27499f84-f5dc-4076-b4bf-cbe6cb01bfc7)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/90

**Comment by eglekaz** [comment]
Fixed with version 0.0.43 of tree-sitter-abl
